modules/govuk_jenkins/manifests/jobs/search_generate_sitemaps.pp

```diff
@@ -0,0 +1,24 @@
+# == Class: govuk_jenkins::jobs::search_generate_sitemaps
+#
+# Create a file on disk that can be parsed by jenkins-job-builder
+#
+class govuk_jenkins::jobs::search_generate_sitemaps{
+  $app_domain = hiera('app_domain')
+
+  $service_description = 'Runs a rake task on Search API that generates the sitemap files'
+  $job_slug = 'search_generate_sitemaps'
+  $job_url = "https://deploy.${app_domain}/job/${job_slug}"
+
+  file { '/etc/jenkins_jobs/jobs/search_generate_sitemaps.yaml':
+    ensure  => present,
+    content => template('govuk_jenkins/jobs/search_generate_sitemaps.yaml.erb'),
+    notify  => Exec['jenkins_jobs_update'],
+  }
+
+  @@icinga::passive_check { "${job_slug}_${::hostname}":
+    service_description => $service_description,
+    host_name           => $::fqdn,
+    freshness_threshold => 86400,
```

Comment:
Maybe 1 day is a bit strict, since if the job is daily it won't necessarily complete again within 24 hours.